### PR TITLE
Fix/added release workflow

### PR DESCRIPTION
**Pull request by abhinav-from-contentstack**
current version : 1.0.7

This PR adds a release workflow, LICENSE updates and a newline at the end of a typings file (to address an eslint error in one of the workflows).

There are no major changes to the code. Hence the package version need not be updated.

### CODE REVIEW

**Comment by netrajpatel** [comment]
@abhinav-from-contentstack 
The GitHub workflow is failing, can you fix that?

**Comment by netrajpatel** [comment]
@abhinav-from-contentstack This workflow is failing, please fix this.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Release contentstack-cli-content-type
```
Comment (netrajpatel) [inline .github/workflows/release.yml:1]: @abhinav-from-contentstack Github actions are failing

**Review by netrajpatel — CHANGES_REQUESTED** [review]

**Comment by abhinav-from-contentstack** [comment]
@netrajpatel Are you referring to the node.js CI workflow? 

It was added by the author of this repository. It tries to build the package using old versions of node, and then runs the test cases. I've never seen it run successfully yet.

Should I fix this?

**Comment by netrajpatel** [comment]
> @netrajpatel Are you referring to the node.js CI workflow?
> 
> It was added by the author of this repository. It tries to build the package using old versions of node, and then runs the test cases. I've never seen it run successfully yet.
> 
> Should I fix this?

Yes please fix this as well.

File: .github/workflows/release.yml (hunk 1 above)
Comment (abhinav-from-contentstack) [reply .github/workflows/release.yml:1]: I've added a fix @netrajpatel , please have a look.

**Review by netrajpatel — CHANGES_REQUESTED** [review]
@abhinav-from-contentstack 
Please add a note for the plugin version update

**Comment by abhinav-from-contentstack** [comment]
@netrajpatel done

**Review by netrajpatel — APPROVED** [review]

**Comment by aman19K** [comment]
Handled in PR #9

**Pull request closed without merging**